addons/web/static/src/scss/layout_background.scss

```diff
@@ -32,7 +32,6 @@
     }
 }
 .o_report_layout_background {
-    background-image: url(/base/static/img/bg_background_template.jpg);
```

Comment:
I think this is not necessary (this would be overridden by the inline CSS for people that do -u) and if done this would cause the background to disappear for people that don't do "-u web"